apps/system/js/app_usage_metrics.js

```diff
@@ -341,9 +374,12 @@
         AUM.REPORT_INTERVAL = result['metrics.appusage.reportInterval'];
         AUM.REPORT_TIMEOUT = result['metrics.appusage.reportTimeout'];
         AUM.RETRY_INTERVAL = result['metrics.appusage.retryInterval'];
+        AUM.ISDOGFOODER = result['debug.performance_data.dogfooding'];
+        (AUM.ISDOGFOODER === '1')? AUM.ISDOGFOODER = true:
+                                   AUM.ISDOGFOODER = false;
```

Comment:
Just curious, why does the value need to be a stringified `'0'` or `'1'`, instead of just `0` or `1`?
 Also, these last 2 lines could be made much less terse:

`AUM.ISDOGFOODER = result['debug.performance_data.dogfooding'] === '1';`